x-pack/plugins/security_solution/public/detections/components/host_isolation/use_host_isolation_action.tsx

```diff
@@ -69,52 +69,56 @@ export const useHostIsolationAction = ({
     [detailsData]
   );
 
-  const {
-    loading: loadingHostIsolationStatus,
-    isIsolated,
-    agentStatus,
-    capabilities,
-  } = useEndpointHostIsolationStatus({
-    agentId,
-    agentType: sentinelOneAgentId ? 'sentinel_one' : 'endpoint',
+  const agentType = sentinelOneAgentId ? 'sentinel_one' : 'endpoint';
+  const isSentinelOneAgent = agentType === 'sentinel_one';
+  const isEndpointAgent = agentType === 'endpoint';
+
+  const agentId = sentinelOneAgentId || endpointAgentId;
+
+  const { data: endpointAgentInfo } = useGetEndpointDetails(agentId, {
+    enabled: isEndpointAgent,
   });
 
-  const { data: sentinelOneAgentData } = useGetSentinelOneAgentStatus([sentinelOneAgentId || '']);
-  const sentinelOneAgentStatus = sentinelOneAgentData?.[`${sentinelOneAgentId}`];
+  const { data: agentStatusData } = useAgentStatus([agentId], agentType, {
+    refetchInterval: 2000,
+    enabled: isEndpointAgent || (isSentinelOneAgent && sentinelOneManualHostActionsEnabled),
```

Comment:
@paul-tavares I think this should also include the `responseActionsSentinelOneV1Enabled` FF. Or maybe it should only use the `responseActionsSentinelOneV1Enabled` FF and not `sentinelOneManualHostActionsEnabled` FF. What do you think? For context, this is the alert details flyout view that shows the Isolation status badge.

Reply:
I think either one is fine - they are both now enabled in Prod, so it does not really matter (right?)